src/Linker/Linker.cpp

```diff
@@ -462,6 +459,32 @@ class LinkerImpl final : public Linker
         return true;
     }
 
+    bool BuildIWD(const std::string& projectName, const ZoneDefinition& zoneDefinition, SearchPaths& assetSearchPaths) const
+    {
+        const fs::path iwdFolderPath(m_args.GetOutputFolderPathForProject(projectName));
+        auto iwdFilePath(iwdFolderPath);
+        const auto iwdFileName = zoneDefinition.m_name + ".iwd";
+        iwdFilePath.append(iwdFileName);
+
+        fs::create_directories(iwdFolderPath);
+
+        const auto iwdWriter = IWDWriter::Create(iwdFilePath.string(), &assetSearchPaths);
+        for (const auto& assetEntry : zoneDefinition.m_assets)
+        {
+            iwdWriter->AddFile(assetEntry.m_asset_name);
+        }
+
+        if (!iwdWriter->Write())
+        {
+            std::cerr << "Writing iwd failed." << std::endl;
+            return false;
+        }
+
+        std::cout << "Created iwd \"" << iwdFilePath.string() << "\"\n";
```

Comment:
Is there a reason for mixing `std::endl` and `\n`? :sweat_smile: 

I know i did it but i try to stick to just `\n` atm since `std::endl` also flushes and that can cost a bit of performance.